documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
+The organisation currently have these classed as _sensitive_
+1. S3 Bucket names <br>
+Bucket names for a public facing endpoint. Bucket names are typically a hash to keep them obscured, putting them in a document would give context and remove the ambiguity.
+2. Individuals names, contact details or any other personally identifiable information.
```

Comment:
We care about all personal information. So, we need to be explicit that things like email addresses and usernames, which will generally have an individual's name attached matter. Also some metadata will fit into this category (for example, files relating to an individual will often reference the individual's name). We should reference the relevant definitions from GDPR and the Data Protection Act 2018 here, as well as linking back to the Data Protection principles which UCFS are developing with us, which provide a more detailed definition of what personal data means in our business context.